plugins/modules/zos_copy.py

```diff
@@ -513,13 +513,18 @@ def copy_to_seq(self, src, temp_path, conv_path, dest, src_ds_type, model_ds=Non
             src_ds_type {str} -- The type of source
         """
         new_src = temp_path or conv_path or src
+        if self.dest_exists:
+            Datasets.delete(dest)
```

Comment:
Does this need to take into account the value of `force`?

Reply:
Actually, the action plug-in determines whether `force` has been specified and whether data should be copied. At this point in the control flow, `force` is either true or destination does not exist. 

If `force` is true then the existing destination can be safely deleted. If the existing destination does not exist, then the if condition will evaluate to false.

Reply:
Perfect, thank you for the clarification!